EIPS/eip-1654.md

```diff
@@ -0,0 +1,107 @@
+---
+eip: 1654
+title: Off-chain dapp-wallet authentication process with contract wallets support
+author: Maor Zamski (@pazams), Christopher Scott (@chrisaxiom)
+discussions-to: https://github.com/ethereum/EIPs/issues/1654
+status: Draft
+type: Meta
+created: 2018-12-12
+requires: 191, 1271
+---
+
+## Simple Summary
+An off-chain dapp-wallet authentication process. The process asserts whether an entity has authorized control (informally, "ownership") over a public Ethereum address using `eth_sign`. Supports both external wallets and contract wallets.
+
+## Definitions
+- `contract wallet` A contract [account](https://github.com/ethereum/wiki/wiki/White-Paper#ethereum-accounts) deployed with the intent to be used as the ownership address for on-chain assets (including ether, ERC-20 tokens, and ERC-721 NFTs). It has the ability to transfer ether or dynamically execute actions on other contracts (acting as the owner of assets controlled by those contracts). Common examples of contract wallets are `multisig wallets` (such as the ones provided by [Gnosis](https://github.com/Gnosis/MultiSigWallet) and [Parity](https://github.com/ConsenSys/MultiSigWallet)) and `identity contracts`, as defined in [ERC-725](https://github.com/ethereum/EIPs/issues/725).
+- `external wallet` An externally owned [account](https://github.com/ethereum/wiki/wiki/White-Paper#ethereum-accounts), controlled by a private key. Currently, most on-chain assets are owned by such accounts. A common example for an external wallet are the wallets generated by MetaMask.
+- `authorized signer` An entity is considered to have authorized control over a wallet if either:
+  - It produced a signature of which the recovered address matches the wallet address.
+  - It produced a signature of which the contract at the wallet address responds with the magic value `0x1626ba7e` to the `isValidSignature()` contract call of [EIP-1271](https://github.com/ethereum/EIPs/blob/master/EIPS/eip-1271.md).
+
```

Comment:
This isn't a section in the [template](https://raw.githubusercontent.com/ethereum/EIPs/master/eip-template.md).  Recommend moving it to the `## Specification`.